paddle/fluid/framework/ir/mkldnn/fc_mkldnn_pass.cc

```diff
@@ -56,13 +56,14 @@ void FCMKLDNNPass::ApplyImpl(ir::Graph* graph) const {
     OpDesc* desc = fc->Op();
     auto dims = fc->inputs[0]->Var()->GetShape();
     auto dim_num = dims.size();
-    bool are_dims_supported = dim_num == 2 || dim_num == 4;
+    bool are_dims_supported = dim_num >= 2 && dim_num <= 4;
     constexpr size_t height_axis = 2;
     constexpr size_t width_axis = 3;
     bool is_size_supported =
         dim_num == 4 ? (dims[width_axis] == 1 && dims[height_axis] == 1) : true;
     if (!are_dims_supported || !is_size_supported) {
-      VLOG(3) << "Do not enable FC MKL-DNN for dimensions different than 2 & 4";
+      VLOG(3)
+          << "Do not enable FC MKL-DNN for dimensions different than 2, 3 & 4";
       VLOG(3) << "Or when width and height are different than one";
```

Comment:
What about joining the two messages into one?
Also:
`width and height are` -> `width or height is`.

Reply:
I just wanted to separate them into  two lines as I thought It'd be cleaner to do that, but can sure merge them. Btw - should've written `when both width and height are equal to one`. Will do in an upcoming commit.

Reply:
Is it fine the way I did it now?